doc/development/deprecations.rst

```diff
@@ -22,15 +28,22 @@ Pending deprecations
   - Deprecated in v0.39
   - Will be removed in v0.40
 
-* The ``decomp_depth`` argument in ``qml.device`` is deprecated. 
+* ``Device``, ``QubitDevice``, and ``QutritDevice`` will no longer be imported top level in v0.40.  They instead
+  we be available as ``qml.devices.LegacyDevice``, ``qml.devices.QubitDevice``, and ``qml.devices.QutritDevice``
+  respectively.
 
-  - Deprecated in v0.38
-  - Will be removed in v0.39
+  - Deprecated top level access in v0.39
+  - Top level access removed in v0.40
 
-* The functions ``qml.qinfo.classical_fisher`` and ``qml.qinfo.quantum_fisher`` are deprecated since they are being migrated
-  to the ``qml.gradients`` module. Therefore, ``qml.gradients.classical_fisher`` and ``qml.gradients.quantum_fisher`` should be used instead.
+* `QNode.gradient_fn` is deprecated. Please use `QNode.diff_method` instead. `QNode.get_gradient_fn` can also be used to
+  process the diff method.
+
+  - Deprecated in v0.39
+  - Will be removed in v0.40
+
+* The ``decomp_depth`` argument in ``qml.device`` is deprecated. 
 
-  - Deprecated and Duplicated in v0.38
+  - Deprecated in v0.38
```

Comment:
Why are these included in this PR? Is there a poor resolution of merge conflict somewhere?

Reply:
Yeah some stuff got duplicated while resolving merge conflicts. Fixed locally now, just waiting for CI to pass before I push the change.